WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -263,7 +331,15 @@ private extension PaymentGatewayAccount {
 }
 
 private enum Constants {
-    static let pluginName = "WooCommerce Payments"
-    static let minimumSupportedWCPayVersion = "3.2.1"
-    static let supportedCountryCodes = ["US"]
+    enum WCPay {
+        static let pluginName = "WooCommerce Payments"
+        static let minimumSupportedPluginVersion = "3.2.1"
+        static let supportedCountryCodes = ["US"]
+    }
+
+    enum Stripe {
+        static let pluginName = "WooCommerce Stripe Gateway"
+        static let minimumSupportedPluginVersion = "5.9.0"
```

Comment:
This will need to be increased once https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2262 is fixed